pkg/ingress/controller.go

```diff
@@ -423,3 +449,26 @@ func (c *Controller) syncSSL(ctx context.Context, ssl *apisixv1.Ssl, event types
 	}
 	return err
 }
+
+func (c *Controller) checkClusterHealthy(ctx context.Context) {
+	for {
+		select {
+		case <-ctx.Done():
+		case <-time.After(5 * time.Second):
+		}
+
+		// Retry three times in a row, and exit if all of them fail.
+		backoff := wait.Backoff{
```

Comment:
Put logic into the `case <-time.After(5 * time.Second)`

Reply:
Why mark this as resolved?  in `case <-ctx.Done()` do not need the logic of health check.